index.html

```diff
@@ -199,5 +191,40 @@ <h1 class="about-title">About me</h1>
                 </div>
         </div>
     </section>
+    <section class="contact">
+        <div class="cont">
+            <h3 class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="https://formspree.io/f/xvoyeaqj" method="post">
+                <ul class="forms">
+                    <li>
+                        <input type="text" name="name" class="input" id="name" placeholder="Full name" required>
+                        <label for="name" class="label">Full name</label>
+                    </li>
+                    <li>
+                        <input type="email" name="email" class="input" id="email" placeholder="Email" required>
+                        <label for="email" class="label">Email</label>
+                    </li>
+                    <li> 
+                        <textarea class="area" rows="5" name="message">Enter text here</textarea>
+                    </li>
+                    <li>
+                        <input type="submit" name="submit" value="Get in touch" class="submit">
+                    </li>
+                </ul>
+            </form>
+        </div>
+        
+    </section>
+    <footer>
+        <div class="last">
+            <ul class="icon">
+                <li><a href="#"><i class="fa-brands fa-linkedin fas"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-twitter fas"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-github fas"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-facebook fas"></i></a></li>
+            </ul>
+            <img src="img/Shape.png">
+        </div>
+    </footer>
```

Comment:
- [ ] Take a look at these 2 images and find what is wrong?... I got it already, you have done fine on adding social media icons 💯 , but they are not corresponding to the ones in the original design regarding the style, what if you try to modify them a bit? 

<img width="763" alt="pt3" src="https://user-images.githubusercontent.com/77843050/205965801-b114684f-f9d9-487b-820a-b161c206092f.png">